GitReleaseManager.yaml

```diff
@@ -0,0 +1,38 @@
+issue-labels-include:
+  - Breaking Change
+  - Deprecate
+  - Feature
+  - C4B Feature
+  - Bug
+  - Improvement
+  - Documentation
+  - Dependency Change
+issue-labels-exclude:
+  - Internal Refactoring
+  - BuildAutomation
+  - NO RELEASE NOTES
+issue-labels-alias:
+  - name:    Documentation
+    header:  Documentation
+    plural:  Documentation
+  - name:    Bug
+    header:  Bug Fix
+    plural:  Bug Fixes
+  - name:    Deprecate
+    header:  Deprecated Feature
+    plural:  Deprecated Features
+create:
+  include-sha-section: true
+  sha-section-heading: "SHA256 Hashes of the release artifacts"
+  sha-section-line-format: "- `{1}\t{0}`"
+close:
+  use-issue-comments: true
+  issue-comment: |-
+    :tada: This issue has been resolved in version {milestone} :tada:
+
+    The release is available on:
+
+    - [GitHub Release](https://github.com/{owner}/{repository}/releases/tag/{milestone})
+    - [PowerShell Gallery](https://www.powershellgallery.com/packages/{repository}/{milestone})

```

Comment:
As the original AU also had a chocolatey package, we should probably have the same and a link to it here.

Reply:
Good catch. I've added this.